front-end/src/App.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
Same here. Why are we disabling it?

Reply:
I used to get an error that goes "React out of scope", I disabled it temporarily and now I've removed it from the files and it works fine.